tests/SDKTest.php

```diff
@@ -124,15 +124,16 @@ class SDKTest extends TestCase
         'ruby' => [
             'class' => 'Appwrite\SDK\Language\Ruby',
             'build' => [
+                'docker run --rm -v $(pwd):/app -w /app/tests/sdks/ruby --env GEM_HOME=/app/vendor ruby:latest bundle install'
             ],
             'envs' => [
-                'ruby-2.7' => 'docker run --rm -v $(pwd):/app -w /app ruby:2.7-alpine ruby tests/languages/ruby/tests.rb',
+                'ruby-2.7' => 'docker run --rm -v $(pwd):/app -w /app --env GEM_HOME=vendor ruby:2.7-alpine ruby tests/languages/ruby/tests.rb',
```

Comment:
It seems like 2.7 tests are working and the rest are not. Maybe related to the fact you only added `--env GEM_HOME=vendor` to the 2.7 test.

Reply:
Pushed the fix, should work now.